### PR TITLE
Update dependencies, fix build, tests and storybook

### PR DESCRIPTION
**Pull request by vwerne**
- Update dependencies
- Fix build
- Fix storybook
- Fix tests
- Move BasicStaticModal to separate folder

### CODE REVIEW

File: packages/lundium/src/components/Tooltip/Tooltip.js [hunk 1]
```diff
@@ -17,6 +18,89 @@ const getPlacement = (placement, isRTL) => {
...
+};
+
+TooltipTrigger.propTypes = {
+	children: PropTypes.node,
+	closeOnReferenceHidden: PropTypes.bool,
+	defaultTooltipShown: PropTypes.bool,
+	getTriggerRef: PropTypes.bool,
+	modifiers: PropTypes.array,
+	onVisibilityChange: PropTypes.func,
+	placement: PropTypes.string,
```
Comment (MaceM8) [inline packages/lundium/src/components/Tooltip/Tooltip.js]: I think we should put a link with possible values: https://popper.js.org/docs/v2/constructors/#options
Or better yet, define our set and allow those values in propTypes as well

**Review by MaceM8 — APPROVED** [review]

File: packages/lundium/package.json [hunk 2]
```diff
@@ -22,29 +22,29 @@
...
-		"rollup-plugin-size-snapshot": "^0.11.0",
-		"rollup-plugin-terser": "^5.2.0"
+		"rollup-plugin-size-snapshot": "^0.12.0",
+		"rollup-plugin-terser": "^7.0.2"
 	},
 	"peerDependencies": {
 		"prop-types": "^15.7.2",
-		"react": "^16.12.0",
-		"react-dom": "^16.12.0"
+		"react": "^17.0.2",
```
Comment (tommmyy) [inline packages/lundium/package.json]: so you say that Lundium will not work in react 16? I think that is not true.

File: packages/theme-basic/package.json [hunk 3]
```diff
@@ -13,26 +13,26 @@
...
+		"sass-loader": "^10.2.0",
 		"ttf-loader": "^1.0.2",
-		"url-loader": "^3.0.0"
+		"url-loader": "^4.1.1"
 	},
 	"peerDependencies": {
 		"prop-types": "^15.7.2",
-		"react": "^16.12.0",
-		"react-dom": "^16.12.0"
+		"react": "^17.0.2",
```
Comment (tommmyy) [inline packages/theme-basic/package.json]: same as above

**Review by tommmyy — CHANGES_REQUESTED** [review]

File: packages/lundium/src/components/Tooltip/Tooltip.js (hunk 1 above)
Comment (vwerne) [reply packages/lundium/src/components/Tooltip/Tooltip.js]: fixed

File: packages/lundium/package.json (hunk 2 above)
Comment (vwerne) [reply packages/lundium/package.json]: fixed

File: packages/theme-basic/package.json (hunk 3 above)
Comment (vwerne) [reply packages/theme-basic/package.json]: fixed

**Review by tommmyy — APPROVED** [review]

**Pull request merged**